plugins/updater/src/updater.rs

```diff
@@ -381,11 +421,11 @@ impl Updater {
                 .to_string()
                 // url::Url automatically url-encodes the path components
                 .replace("%7B%7Bcurrent_version%7D%7D", &encoded_version)
-                .replace("%7B%7Btarget%7D%7D", &self.target)
+                .replace("%7B%7Btarget%7D%7D", target)
```

Comment:
```suggestion
                .replace("%7B%7Btarget%7D%7D", target)
                .replace("%7B%7Binstaller%7D%7D", installer.name())
```
needs the let installer moved above this of course. ignore this if this was planned for another pr i guess

Reply:
I prefer to leave it for another PR

Reply:
hmm okay